src/frequenz/sdk/actor/power_distributing/_battery_pool_status.py

```diff
@@ -121,7 +124,7 @@ def __init__(
             **receivers,
         )
 
-        self._task = asyncio.create_task(self._run())
+        self._task = asyncio.create_task(self._run(battery_status_sender))
```

Comment:
Because it is not needed as an instance variable. 
There is already lots of instance variables.

Reply:
For what's worth, I agree with Sahas that it would make sense to make it an instance variable, it seems like something very inherent to the class. If at some point you would want to update the sender, it won't be possible if you pass it around. But for me is minor and I'm fine to leave it as it is too if you feel it shouldn't be an instance variable, in the extremely rare event that we want to give the option to replace the sender in the future we can change it then.